share/smack/top.py

```diff
@@ -610,13 +589,15 @@ def annotate_bpl(args):
 
 
 def property_selection(args):
```

Comment:
Maybe this function is better named as `memory_safety_property_selection`

Reply:
Good point. Done.